sdks/python/apache_beam/typehints/opcodes.py

```diff
@@ -490,6 +490,13 @@ def load_fast(state, arg):
   state.stack.append(state.vars[arg])
 
 
+def load_fast_load_fast(state, arg):
+  arg1 = arg >> 4
+  arg2 = arg & 15
```

Comment:
consider including links to opcode defs, like https://docs.python.org/3/library/dis.html#opcode-LOAD_FAST_LOAD_FAST, so it's easier to reason about the referenced constants.

Reply:
I'll file an issue and take that on as a follow-up, the documentation around this piece of the type checking infra needs to be updated.

Reply:
Filed as #34930